### ISSUE TITLE
Multiple servers running at once

### ISSUE THREAD

**Issue by redoper1** [rpcm-org/docker-rpcm#7]
Please add possibility to change port for FiveM server, so we can run multiple instances at once.

**Comment by celestian** [comment]
I was thinking about it. The port is written in:
1. `server.cfg`
1. `docker-compose.yaml`

I hope that `docker-compose.yaml` will consume port from env. variable. Then, it will be easy.

**Comment by celestian** [comment]
I am little afraid there will be issue with inner network. If you want to have two or more servers running in the same time -- maybe the best solution is to create whole ecosystem again. I can prepare script which does it for you automatically. But in the evening, not now. :-)

**Comment by celestian** [comment]
Or, and this is maybe what we are looking for, I can create static docker-compose.yaml files for testing purposes. Whole ecosystem will be copied, but the docker images can be created from different branch of RPCM project. So, you can prepare testing environment for testing given branch of rpcm framework very easily. What is needed? Just account on `dockerhub.com`, so I can add you to our docker hub organization.

**Comment by redoper1** [comment]
I think for us it's right now not needed, it was more for public in later phase.
I know that server port is written in these two files and I was thinking if the `docker-compose.yaml` file will consume this '_port variable_'.

**Comment by celestian** [comment]
**SOLUTION**
Due to complexity of our docker-compose ecosystem, it is not possible to run multiple ecosystem in parallel. But you can modify manually docker-compose and `.rpcm.env` file in way, that the second fivem server will be operate on the other port. You should copy `RPCM_DB_NAME` and create clone with different `DB_NAME` for the second server.

The universal solution isn't so easy so we recommend to do the manual changes described above and let it to benevolent reader.
